actuator-firmware/src/uavcan/Command_handler.cpp

```diff
@@ -0,0 +1,35 @@
+#include "Command_handler.hpp"
+#include "servo.h"
+
+#include <error/error.h>
+
+#include <cvra/actuator/Command.hpp>
+
+#include "uavcan/build_config.hpp"
+
+int Command_handler_start(uavcan::INode& node)
+{
+    int ret;
+    static uavcan::Subscriber<cvra::actuator::Command> sub(node);
+    ret = sub.start(
```

Comment:
It's good practice to initialize local variables at declaration. Or in this case, return directly.
And always use const when possible.

Reply:
done.